modules/wmco-prerequisites.adoc

```diff
@@ -18,18 +18,18 @@ The following table lists the link:https://docs.microsoft.com/en-us/windows/rele
 |Supported Windows Server version
 
 |Amazon Web Services (AWS)
-|Windows Server 2019, version 1809 Long-Term Servicing Channel (LTSC)
+|Windows Server 2019, version 1809
```

Comment:
These table entries are all fragments, so none of them should have terminating periods.